mala/interfaces/ase_calculator_uncertainty.py

```diff
@@ -0,0 +1,157 @@
+"""ASE calculator for MALA predictions."""
+
+from ase.calculators.calculator import Calculator, all_changes
+import numpy as np
+
+from mala import Parameters, Network, DataHandler, LDOS, MultiPredictor
+from mala.common.parallelizer import get_rank, get_comm, barrier
+from mala.interfaces.ase_calculator import MALA
+
+
+class MALAUncertainty(MALA):
```

Comment:
We plan to implement other UQ methods and I need to think about an inferface for that, but we can't steal the generic name `MALAUncertainty` here :)

I suggest `MALASimpleEnsemble`. See also the other suggested name change below: `MultiPredictor` -> `SimpleEnesmblePredictor`.

Reply:
Yeah, good point. I will change the name.